kratos/utilities/specifications_utilities.h

```diff
@@ -52,6 +52,26 @@ namespace Kratos
  */
 namespace SpecificationsUtilities
 {
+    /**
+     * @brief This enum defines a "hash" used to identify if implicit/explicit or static time integration is considered
+     */
+    enum class TimeIntegration
+    {
+        Static   = 0,
+        Implicit = 1,
+        Explicit = 2
+    };
+
+    /**
+     * @brief This enum defines a "hash" used to identify if Lagrangian/Eulerian or ALE framework is considered
+     */
+    enum class Framework
+    {
+        Lagrangian = 0,
+        Euler      = 1,
+        ALE        = 2
+    };
+
```

Comment:
Should we add the Updated Lagrangian to the list too?

Reply:
Nope, Updated Lagrangian is still Lagrangian. The point of Lagrangian is communicate to the solver that the nodes of these elements/conditions require to be moved after solving the step

Reply:
Well, fort now this just tells the framework, for now it is not taken into account for nothing

Reply:
Your argument makes sense to me (the updated is still Lagrangian).